frontend/src/components/HealthMetrics.tsx

```diff
@@ -3,32 +3,24 @@ import {
   faCodeCommit,
   faCodeFork,
   faCodePullRequest,
-  faHeart,
   faStar,
   faTag,
 } from '@fortawesome/free-solid-svg-icons'
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import React from 'react'
 import type { HealthMetricsProps } from 'types/healthMetrics'
 import GradientRadialChart from 'components/GradientRadialChart'
 import LineChart from 'components/LineChart'
 
 const HealthMetrics: React.FC<{ data: HealthMetricsProps[] }> = ({ data }) => {
   const openIssuesCountArray = data.map((item) => item.openIssuesCount)
-  const labels = data.map((item, index) => `Day ${index + 1}`)
+  const labels = data.map((item) => {
```

Comment:
Thanks for adding dates, but they are showing backwards  😅 I think it should be Jun 20th, and then Jun 21st. Could you order them by date (preferably on the backend)?
![Screenshot 2025-06-21 at 5 49 53 PM](https://github.com/user-attachments/assets/5240d918-ea5d-469f-9a2d-aca505e540e5)